src/components/aboutUs/__tests__/StorySection.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { shallow } from "enzyme";
+import StorySection from "../StorySection";
+
+describe("Test StorySection component", () => {
+    it("should be able to render <StorySection />", () => {
+        shallow(<StorySection />);
```

Comment:
Now actual testing in this file. Please add some test cases.

Reply:
Thanks for the suggestions, have been modified and tested in local files.